atd-toolbox/data_model/python/sql_templates/crashes_list_view.sql

```diff
@@ -0,0 +1,148 @@
+create or replace view crashes_list as with injury_severities as (
+    select
+        id,
+        unit_id,
+        est_comp_cost_crash_based,
+        case
+            when (prsn_injry_sev_id = 9) then 1
+            else 0
+        end as unkn_injry,
+        case
+            when (prsn_injry_sev_id = 1) then 1
+            else 0
+        end as sus_serious_injry,
+        case
+            when (prsn_injry_sev_id = 2) then 1
+            else 0
+        end as nonincap_injry,
+        case
+            when (prsn_injry_sev_id = 3) then 1
+            else 0
+        end as poss_injry,
+        case
+            when (prsn_injry_sev_id = 4) then 1
+            else 0
+        end as fatal_injury,
+        case
+            when (prsn_injry_sev_id = 5) then 1
+            else 0
+        end as non_injry
+    from
+        public.people
+),
+
+cris_fatal_injury_counts as (
+    select
+        crashes.crash_id,
+        count(crashes.crash_id) as cris_fatality_cnt
+    from public.crashes_cris as crashes
+    left join public.units_cris as units on crashes.crash_id = units.crash_id
+    left join public.people_cris as people on units.id = people.unit_id
+    where people.prsn_injry_sev_id = 4
+    group by crashes.crash_id
+),
+
+injury_counts as (
+    select
+        crashes.crash_id,
+        sum(injury_severities.unkn_injry) as unkn_injry_cnt,
+        sum(
+            injury_severities.nonincap_injry
+        ) as nonincap_injry_cnt,
+        sum(injury_severities.poss_injry) as poss_injry_cnt,
+        sum(injury_severities.non_injry) as non_injry_cnt,
+        sum(injury_severities.sus_serious_injry) as sus_serious_injry_cnt,
+        sum(injury_severities.fatal_injury) as atd_fatality_cnt,
```

Comment:
its a bit confusing to me that `atd_fatality_cnt` is the only one prepended with `atd`, but i know thats because there is also the `cris_fatality_cnt` and the `apd_fatality_cnt`